src/test/java/Homework17.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+public class Homework17 extends BaseTest {
+
+@Test
+    public void addSongToPlaylist() throws InterruptedException {
+    navigateToPage();
+    login("demo@class.com", "te$t$tudent");
+    Thread.sleep(2000);
+    grabASong();
+}
+
+public void grabASong() {
+    WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
+    WebElement playlist = driver.findElement (By.xpath("//section[@id='playlists']//li[3]"));
+
+    Actions acts = new Actions(driver);
+    acts.clickAndHold(song)
+            .release(playlist)
+            .build()
+            .perform();
+
+    playlist.click();
+   // acts.dragAndDrop(song, playlist).build().perform();
+
+
+}
+
+}
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)